include/api.php

```diff
@@ -2405,8 +2418,8 @@ function api_convert_item($item)
 		$statustext = trim($statustitle."\n\n".$statusbody);
 	}
 
-	if (($item["network"] == NETWORK_FEED) && (strlen($statustext)> 1000)) {
-		$statustext = substr($statustext, 0, 1000)."... \n".$item["plink"];
+	if ((defaults($item, 'network', Protocol::PHANTOM) == Protocol::FEED) && (strlen($statustext)> 1000)) {
```

Comment:
I guess this is done at many more places, so I won't comment all. Also the network field is always set.

Reply:
Unfortunately we can't guarantee that just from this function alone. Until we can guarantee the field is always set, I'm keen on leaving these checks.